### ISSUE TITLE
DEV: Vendor Go Live broken and duplicated with verified

### ISSUE THREAD

**Issue by apowelka** [massenergize/frontend-admin#229]
In dev, the Vendor go live button is broken - toggling yes/no does not change the vendor displayed on the portal. ALSO, the functionality of go live is duplicated in the "have you verified this vendor?" If I click yes to verify but no to go live, it still displays.

**Comment by Davidjwkinghorn** [comment]
Needs verification that this issue no longer exists.

**Comment by frimpongopoku** [comment]
I dont think this is an issue anymore... Since the current all vendors page does not have any live/unlive toggles 🤔 

![Image](https://user-images.githubusercontent.com/26961591/171607675-07c70271-6e68-4a56-8ec7-3e7d86fb2484.png)
